.github/workflows/tag.yaml

```diff
@@ -0,0 +1,28 @@
+name: create_tag
+on: 
+  pull_request:
+    types: [closed]
+
+permissions:
+  contents: write
+
+jobs:
+  create_tag:
+    name: "Create tag"
+    if: github.event.pull_request.merged == true && contains(github.event.pull_request.labels.*.name, 'release-pr') && startsWith(github.event.pull_request.title, format('chore{0} Prepare', ':'))
```

Comment:
can we restrict this more?

Reply:
thanks! with the current config, anyone in the opa org will be able to label and title a PR accordingly will be able to create a tag